packages/stepper/src/Stepper.spec.tsx

```diff
@@ -54,7 +58,7 @@ function assertVisibleSteps({
   );
 
   if (expectedSteps.includes('Previous steps')) {
-    expect(visiblePreviousStepsElement).not.toBeNull();
+    expect(visiblePreviousStepsElement).toBeVisible();
```

Comment:
Just out of curiosity, why the change?

Reply:
Making this consistent everywhere based on the discussion in this thread
https://github.com/mongodb/leafygreen-ui/pull/471/files/840f8c5dd7dc762382d69676dbdad60b74a1000e#diff-832b8f9837f5879d08b9fddbcc69af38

Reply:
Can't see that thread gh is erroring :/

Reply:
oops, updated the link